tests/testdata/buf.build/library/connect-go/v0.4.0/petapis/plugin.sum

```diff
@@ -0,0 +1 @@
+h1:LNnCcjOu8kcVABekU/5d8OgdAwuFzE3MFcMoTv2iFXI=
```

Comment:
Ran `make test` locally to generate these files. There are no differences between the v0.3.0 and v0.4.0 versions of the generated code.